clang/include/clang/Basic/DiagnosticDriverKinds.td

```diff
@@ -442,6 +442,10 @@ def warn_drv_deprecated_arg : Warning<
 def warn_drv_deprecated_arg_no_relaxed_template_template_args : Warning<
   "argument '-fno-relaxed-template-template-args' is deprecated">,
   InGroup<DeprecatedNoRelaxedTemplateTemplateArgs>;
+def warn_drv_deprecated_arg_ofast : Warning<
+  "argument '-Ofast' is deprecated; use '-O3 -ffast math' for the same behavior,"
```

Comment:
Missing dash between ffast and math. 
Here and several times below.

Reply:
Thank you for letting me know. Fixed in 5a45fed188dce2ae6c4da23ba61c9b8df87f08f4